middleware/static/config.go

```diff
@@ -0,0 +1,93 @@
+package static
+
+import (
+	"io/fs"
+	"time"
+
+	"github.com/gofiber/fiber/v3"
+)
+
+// Config defines the config for middleware.
+type Config struct {
```

Comment:
should we add something for SPA´s ?
do they work directly ? can we have a test and readme example for theses?

until now it was 

https://github.com/gofiber/recipes/blob/aa602c67e82007838a33ea8a99effcfd37bba8c4/spa/main.go#L19-L23

or for embeded files
```go
    app.Use("/", filesystem.New(filesystem.Config{
        Root:       http.FS(f),
        PathPrefix: "ui/build",
    }))

    app.Use("*", func(ctx *fiber.Ctx) error {
        return filesystem.SendFile(ctx, http.FS(f), "ui/build/index.html")
    })

```

Reply:
pls also check the conversation from fenny (april) -> https://discord.com/channels/704680098577514527/704683224147689533/1228351805524672542

Reply:
Don't we have c.SendFile for that?

Reply:
Yeah good idea will expiose notfound

Reply:
> Don't we have c.SendFile for that?

would be cool, if it gets easier for the consumer 
there were lot of questions regarding the usage with SPA static content

would be nice when we cover these cases with the new implementation 
and also less questions for this topic